src/networking.c

```diff
@@ -2508,7 +2508,9 @@ int processMultibulkBuffer(client *c) {
         } else {
             /* Check if we have space in argv, grow if needed */
             if (c->argc >= c->argv_len) {
-                c->argv_len = min(c->argv_len < INT_MAX/2 ? c->argv_len*2 : INT_MAX, c->argc+c->multibulklen);
+                c->argv_len = c->argv_len > 0 ? c->argv_len*2 : 1; /* argv_len might be 0 if argv was
+                                                                    * released due to client timeout */
```

Comment:
what is `client timeout`？we will close client if reaching client timeout

Reply:
it should be `idle` not `timeout`.
Updated the comment.